test/api_bridge_test.rb

```diff
@@ -27,7 +27,7 @@ def before_setup
     facets = ["creator.name", "date.year", "subcategory"]
     assert_equal facets, @api.app_facets
     assert_equal(
-      { "num" => 5, "sort" => "title|asc", "facet" => facets },
+      {"num"=>5, "sort"=>"title|asc", "f"=>["collection|letters"], "facet"=>["creator.name", "date.year", "subcategory"]},
```

Comment:
Would you mind restoring the whitespace? :pray: It looks like a really long line that should be split across lines too. Style guide conventions

Reply:
this is partly a casualty of me not being able to boot up git gui today :'(

Reply:
:fist_raised: Bad Jess's computer! It's going to have time in the corner later